include/proto.h

```diff
@@ -491,6 +492,16 @@ enum {
    SL_FLICKER_OFF      = 2,         // Turn flickering off
 };
 
+/* Special item types */
+enum special_type {
```

Comment:
To avoid having to use "enum" everywhere, typically this is done:

typedef enum { ... } special_type;

Reply:
Changed as suggested.